package.json

```diff
@@ -23,31 +23,31 @@
     "projen": "npx projen"
   },
   "devDependencies": {
-    "@types/jest": "^27.5.1",
+    "@types/jest": "^27.5.2",
     "@types/node": "^12",
     "@typescript-eslint/eslint-plugin": "^5",
     "@typescript-eslint/parser": "^5",
     "eslint": "^8",
-    "eslint-import-resolver-node": "^0.3.6",
+    "eslint-import-resolver-node": "^0.3.7",
     "eslint-import-resolver-typescript": "^2.7.1",
-    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-import": "^2.27.5",
     "jest": "^27.5.1",
     "jest-junit": "^13",
     "json-schema": "^0.4.0",
     "npm-check-updates": "^12",
     "projen": "^0.52.37",
     "standard-version": "^9",
     "ts-jest": "^27.1.5",
-    "typescript": "^4.7.2"
+    "typescript": "^4.9.5"
   },
   "dependencies": {
-    "@aws-sdk/client-dynamodb": "^3.100.0",
-    "@aws-sdk/lib-dynamodb": "^3.100.0",
-    "@aws-sdk/smithy-client": "^3.99.0",
-    "@aws-sdk/types": "^3.78.0",
-    "@aws-sdk/util-dynamodb": "^3.100.0",
-    "@types/aws-lambda": "^8.10.97",
-    "aws-sdk": "^2.1145.0"
+    "@aws-sdk/client-dynamodb": "^3.378.0",
+    "@aws-sdk/lib-dynamodb": "^3.378.0",
+    "@aws-sdk/smithy-client": "^3.370.0",
```

Comment:
According to the message, I think this will need a replacement for @smithy/smithy-client

Reply:
I took a look at fixing this PR. Ran into some critical type error when upgrading to Ts 5 and latest smithy. Needs some dedicated attention. I'll try and find some time this weekend to fix!

Reply:
no need to stress it, I'm glad that it will eventually work.